### ISSUE TITLE
20221226 react myapp Cannot read properties of undefined (reading 'basic')

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#5293]
I removed node_modules and recompiled. It did say on the build:

```txt
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
```

The screen is blank and the first error is:

```txt
Uncaught TypeError: Cannot read properties of undefined (reading 'basic')
    at new PanelLayout (panel.tsx:27:1)
    at new <anonymous> (form.tsx:22:1)
    at LayoutWrapper.getLayoutForSchema (layouts.ts:54:1)
    at simulation.js:100:1
    at Array.map (<anonymous>)
    at SimulationRoot (simulation.js:96:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
```

jspec and genesis seem to work.

**Comment by robnagler** [comment]
Also package-lock.json and package.json were overwritten with the npm install. I will checkout the old ones, but I wouldn't expect them to be overwritten just by running `sirepo service http`.

**Comment by garsuga** [comment]
Which app are you getting the error in?

**Comment by robnagler** [comment]
myapp

**Comment by garsuga** [comment]
I cannot recreate this, could this be another issue with old schema being held in an installed version of sirepo?

**Comment by moellep** [comment]
I think this is because the react-json/myapp-schema.json hadn't been updated with the new format. I added this in this pull request: https://github.com/radiasoft/sirepo/pull/5343/commits/578110a2e259f6ac7b5d959a622da484ebd85589

**Comment by garsuga** [comment]
Closed as fixed in #5343
